### ISSUE TITLE
Websocket Client can't handle KeepAlive message

### ISSUE THREAD

**Issue by alexvelea** [99designs/gqlgen#2291]
### What happened?
I created a stress test for my server similar to the [chat example](https://github.com/99designs/gqlgen/blob/master/_examples/chat/chat_test.go#L14). The stress test is working great until it reaches the `10s` mark when the KeepAlive message is being sent. The 10s is the default KA interval for the server and I'd like to not mess with that if possible. 

When the WebSocket Client receives a KeepAlive message it can't handle it and it just crashes with the following error:
```
expected data message, got client.operationMessage{Payload:json.RawMessage(nil), ID:"", Type:"ka"}
```

### What did you expect?
The KeepAlive message to be abstracted away from user applications. The WebSocket client comes by default with `gqlgen` or transport layer itself to handle the KA logic.

### Background
I've looked into older PRs and there's this one, #2209 which actually implements support for KeepAlive messages in the websocket client in commit 670cf22272b490005d46dc2bee1634de1cd06d68 but it's later reverted in d8309fd2b2e6a0dabb79a502f8a59d48930d3b7e. 

I totally agree that the support for KA messages in WS Client wouldn't've fixed that particular problem but I still argue it's a good feature to have.

### versions
 - `go run github.com/99designs/gqlgen version` 
   - `v0.17.2`
 - `go version`
    - `go version go1.18.3 darwin/arm64`

**Comment by frederikhors** [comment]
Can you PR?

**Comment by alexvelea** [comment]
@frederikhors opened PR #2293

**Comment by frederikhors** [comment]
Technically this is not closed yet.

**Comment by alexvelea** [comment]
Fixed in #2293
